documentation/tutorial/02-environment.md

```diff
@@ -13,75 +13,68 @@ Before we start, make sure your machine meets the following requirements:
   [Multipass](https://multipass.run/docs). A simple set of instructions to set
   up Multipass can be found
   [here](https://juju.is/docs/sdk/set-up-your-development-environment#heading--set-up-an-ubuntu-vm-with-multipass).
-  Please keep in mind that the instructions in this tutorial have not been
-  tested with Multipass.
 - 8GB of RAM.
 - 2 CPU threads.
 - At least 20GB of available storage.
 - Access to the internet for downloading the required snaps and charms.
 
-## Install MicroK8s
+## Install Canonical Kubernetes
 
-On your machine, install and configure MicroK8s:
+On your machine, install and configure Canonical Kubernetes (K8s):
 
 ```bash
-# Install Microk8s from snap:
-sudo snap install microk8s --channel 1.25-strict/stable
+sudo snap install k8s --classic
 
-# Add your user to the MicroK8s group:
-sudo usermod -a -G snap_microk8s $USER
-
-# Give your user permissions to read the ~/.kube directory:
-sudo chown -f -R $USER ~/.kube
-
-# Create the 'microk8s' group:
-newgrp snap_microk8s
-
-# Enable the necessary MicroK8s addons:
-sudo microk8s enable hostpath-storage dns
-
-# Set up a short alias for the Kubernetes CLI:
-sudo snap alias microk8s.kubectl kubectl
+sudo k8s bootstrap
+```
+Wait until the cluster is in `Ready` state. You can check it as follows:
```

Comment:
nit: until the `k8s` cluster (we have many different "clusters", database/node/k8s/temporal. good to be explicit)